tests/frontend/unit/ParticipationPhases.spec.js

```diff
@@ -1,53 +1,47 @@
-import { createLocalVue } from '@vue/test-utils'
 import ParticipationPhases from '@DpJs/components/procedure/basicSettings/ParticipationPhases'
 import shallowMountWithGlobalMocks from '@DpJs/VueConfigLocal'
 
 describe('ParticipationPhases', () => {
-  const componentsPropsData = {
-    autoswitchHint: 'Here comes a tooltip Hint',
-    fieldName: 'some_name',
-    helpText: 'Some Help Text',
-    labelText: 'I am a Label',
-    participationPhases: ['first', 'second', 'third'],
-    permissionMessage: 'Some Message',
-    phaseOptions: [{
-      label: 'My Label C',
-      permissionset: 'read',
-      value: 'first'
-    }, {
-      label: 'My Label A',
-      permissionset: 'write',
-      value: 'fourth'
-    }, {
-      label: 'My Label B',
-      permissionset: 'hidden',
-      value: 'fifth'
-    }],
-    initSelectedPhase: 'first',
-    iterator: {
-      label: 'Iterator Label',
-      name: 'iterator_input_name',
-      tooltip: 'Tooltip text for the iterator',
-      value: '1'
-    }
-  }
-
-  it('should be an object', () => {
-    expect(typeof ParticipationPhases).toBe('object')
+  let wrapper
+
+  beforeEach(() => {
+    wrapper = shallowMountWithGlobalMocks(ParticipationPhases, {
+      propsData: {
+        autoswitchHint: 'Here comes a tooltip Hint',
+        fieldName: 'some_name',
+        helpText: 'Some Help Text',
+        labelText: 'I am a Label',
+        participationPhases: ['first', 'second', 'third'],
+        permissionMessage: 'Some Message',
+        phaseOptions: [{
+          label: 'My Label C',
+          permissionset: 'read',
+          value: 'first'
+        }, {
+          label: 'My Label A',
+          permissionset: 'write',
+          value: 'fourth'
+        }, {
+          label: 'My Label B',
+          permissionset: 'hidden',
+          value: 'fifth'
+        }],
+        initSelectedPhase: 'first',
+        iterator: {
+          label: 'Iterator Label',
+          name: 'iterator_input_name',
+          tooltip: 'Tooltip text for the iterator',
+          value: '1'
+        }
+      }
+    })
   })
 
-  it('should be named ParticipationPhases', () => {
-    expect(ParticipationPhases.name).toBe('ParticipationPhases')
+  it('should be instantiated', () => {
+    expect(wrapper).toBeTruthy()
```

Comment:
why this change?

Reply:
Good point. I now removed testing for object, too (like the test for name), because testing for implementation details is considered bad practice and provides little to no value. Name() is also deprecated in vue test utils and will be removed soon (see https://v1.test-utils.vuejs.org/upgrading-to-v1/#name). 
But I also saw that we test name and object very often. Should we discuss this with the team?